src/components/Staking/StakingHierarchy.tsx

```diff
@@ -287,9 +290,11 @@ const StakingHierarchy: React.FC<IProps> = () => {
           <SoloGlyph />
         </Glyph>
         <Content>
-          <p>
-            <Translation id="page-staking-hierarchy-solo-p1" />
-          </p>
+          <p
+            dangerouslySetInnerHTML={{
+              __html: t("page-staking-hierarchy-solo-p1"),
+            }}
+          />
```

Comment:
This is a temporary implementation. In the next iteration we are planning in using the following approach: https://react.i18next.com/latest/trans-component#using-with-react-components where we will also need to modify our translation files.